docs/architecture/managers/blocks-manager.md

```diff
@@ -0,0 +1,3 @@
+# Blocks Manager
+
+The __blocks manager__ is the actor in charge of managing the blocks of Witnet blockchain that resides in the node...
```

Comment:
This is WIP, right? :grin:

Reply:
As discussed, we should put here some basic description of the responsibilities of this manager. Issue #123 may already have what we need.